client/src/js/components/bhModalNotify/bhModalNotify.html

```diff
@@ -0,0 +1,21 @@
+<style>
+  .bh-modal-notify {
+    position: fixed;
+    top: 0px;
+    left: 0px;
+    width: 100%;
+    height: 53px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 5px 10px;
+  }
+  .bh-modal-notify > i {
+    cursor: pointer;
+  }
+</style>
+
+<div ng-if="$ctrl.visible" class="bh-modal-notify {{$ctrl.style}}">
```

Comment:
It's more the angularjs way to use `ngClass`.  Could you do that here?

Reply:
I was thinking about that but ngClass allow value in this format : 
```html
ng-class="{'the_css_class': javascript_variable }"
```

Here in the `bhModalNotify` is not known in advance